book/chapters/chapter13/beyond_regression_and_classification.qmd

```diff
@@ -128,12 +128,14 @@ By using `po("learner_cv")` for internal resampling and `po("tunethreshold")` to
 ## Survival Analysis {#sec-survival}
 
 `r index("Survival analysis")` is a field of statistics concerned with trying to predict/estimate the time until an event takes place.
-This predictive problem is unique as survival models are trained and tested on data that may include 'censoring', which occurs when the event of interest does *not* take place.
+This predictive problem is unique because survival models are trained and tested on data that may include censoring, which occurs when the exact event time is not observed.
```

Comment:
```suggestion
This predictive problem is unique because survival models are trained and tested on data that may include censoring, which occurs when the exact event time is not observed for some individuals.
```